03_prompt_tuning/spot_check.py

```diff
@@ -109,13 +98,28 @@ def get_company_info(example):
     return prompt
 
 
+class LaminiModelStage(GenerationNode):
+    def __init__(self, model_name="meta-llama/Meta-Llama-3-8B-Instruct",):
+        super().__init__(
+            model_name=model_name,
+            max_new_tokens=150,
+        )
+        self.model_name = model_name
```

Comment:
don't need self.model_name here

we can remove this __init__ and just do the model_name, max_new_tokens init at the instantiation like this
 
self.model_stage = LaminiModelStage(
            "meta-llama/Meta-Llama-3-8B-Instruct", max_new_tokens=150
        )

Reply:
Done.